python/pyspark/ml/torch/distributor.py

```diff
@@ -501,26 +537,42 @@ def _run_distributed_training(
     ) -> Optional[Any]:
         if not framework_wrapper_fn:
             raise RuntimeError("Unknown combination of parameters")
+
+        log_streaming_server = LogStreamingServer()
+        self.driver_address = get_driver_host(self.sc)
+        log_streaming_server.start()
```

Comment:
We should ensure shutting down the server when distributed training completes or raises error.